catalog/ui/catalog-ui-search/src/main/webapp/component/list-editor/list-editor.view.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, require, module*/
+var Marionette = require('marionette');
+var _ = require('underscore');
```

Comment:
Should we try to standardize whether `_` is `lodash` or `underscore` moving forward in new files?

Reply:
I think there should be a tech debt ticket to go through and try and move us completely to lodash, but I think we've run into issues removing underscore in the past.  Lodash is more maintained and developed, and also supports per method packages which helps out with reducing UI size.